packages/effects-core/src/plugins/animation-graph/graph-node.ts

```
@@ -0,0 +1,174 @@
+
+export class GraphNode {
+  /**
+   * @internal
+   */
+  asset: GraphNodeData;
+
+  private initializationCount = 0;
+  private lastUpdateID = -1;
+
+  constructor () {
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)